File: ExemploSwingMVC/src/main/java/com/mycompany/exemploswingmvc/model/Usuario.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.exemploswingmvc.model;


public class Usuario {
    
    private int id;
    private String cpf;
    private String nome;
    private TipoUsuario tipoUsuario;
    
    public Usuario(){
        tipoUsuario = new TipoUsuario();
    }
    
    public int getId() {
        return id;
    }
    
    public void setId(int id){
        this.id = id;
    }
    
    public String getCpf() {
        return cpf;
    }
    
    public void setCpf(String cpf) {
        this.cpf = cpf;
    }
    
    public String getNome() {
        return nome;
    }
    
    public void setNome(String nome) {
        this.nome = nome;
    }
    
    public TipoUsuario getTipoUsuario() {
        return tipoUsuario;
    }
    
    public void setTipoUsuario(TipoUsuario tipousuario) {
        this.tipoUsuario = tipoUsuario;
    }   
}

File: ExemploMVCWebForms/ExemploMVCWebForms/ExemploMVCWebForms/model/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ExemploMVCWebForms.model
{
    public class Usuario
    {
        private int id;
        private string nome;
        private string cpf;
        private TipoUsuario tipoUsuario;
        
        public Usuario()
        {
            tipoUsuario = new TipoUsuario();
        }
        public int Id
        {
            get => id;
            set => id = value;
        }
        public string Nome
        {
            get => nome;
            set => nome = value;
        }
        public string Cpf
        {
            get => cpf;
            set => cpf = value;
        }
        public TipoUsuario TipoUsuario
        {
            get => tipoUsuario;
            set => tipoUsuario = value;
        }
    }
}